build.sh

```diff
@@ -49,3 +50,12 @@ case "$os" in
     mkdir -p ${OUTDIR}
     cp -f build/bin/Datadog.Trace.ClrProfiler.Native.${SUFIX} ${OUTDIR}/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.${SUFIX}
 esac
+
+# build Managed
+cd $DIR
+OUTDIR="$( pwd )/src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64"
+
+for framework in ${SDK_TARGET_FRAMEWORKS} ; do
+    mkdir -p "$OUTDIR/$framework"
+    dotnet publish -f $framework -c ${BUILD_TYPE} src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.csproj -o "$OUTDIR/$framework"
+done
```

Comment:
it is always good to end with a new line when dealing with *nix stuff 😉 
Reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline